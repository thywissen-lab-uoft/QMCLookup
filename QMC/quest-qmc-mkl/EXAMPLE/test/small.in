# ==========================
# Output file name header
# ==========================

ofile  = small

# ==========================
# lattice dimension
# ==========================

nx     = 6
ny     = 6

# ==========================
# Hubbard model
# ==========================

U      = 10
t_up   = 0.0
t_dn   = 0.0
mu_up  = 5.0
mu_dn  = 5.0
L      = 200
dtau   = 0.01 
HSF    = -1

# ==========================
# Met's algorithm
# ==========================

nwarm  = 1000
npass  = 1000
ntry   = 0
tdm    = 0

# ==========================
# Measurements
# ==========================

nbin   = 5
tausk  = 1  
seed   = 1234567

# ==========================
# numerical
# ==========================

north  = 5 
nwrap  = 15
fixwrap = 1
errrate = 0.001
difflim = 0.00001


