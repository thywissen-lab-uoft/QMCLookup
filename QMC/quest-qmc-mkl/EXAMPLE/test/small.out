 2D Periodic Lattice; Nx=  6; Ny=  6; total sites=   36                          
                          U :           10.000000
                       t_up :            0.000000
                       t_dn :            0.000000
                      mu_up :            5.000000
                      mu_dn :            5.000000
             Time slice - L :          200
            Number of sites :           36
                       dtau :            0.010000
                       beta :            2.000000
     Number of warmup sweep :         1000
Number of measurement sweep :         1000
   Frequency of measurement :            1
                Random seed :      1234567
 Frequency of recomputing G :           15
Global move number of sites :            0
               Accept count :      7301222
               Reject count :      7012477
    Approximate accept rate :            0.510086
                      gamma :            0.797390
           Type of matrix B : Dense MatB  
         Type of matrix HSF : discrete
============================================================================
 Sign of equal time measurements:
                   Avg sign :   0.10000000E+01 +-  0.00000000E+00
                Avg up sign :   0.10000000E+01 +-  0.00000000E+00
                Avg dn sign :   0.10000000E+01 +-  0.00000000E+00
============================================================================
 Equal time measurements:
          Up spin occupancy :   0.67611274E+00 +-  0.63497263E-02
        Down spin occupancy :   0.66049132E+00 +-  0.11324844E-01
             <U*N_up*N_dn>  :   0.33660405E+01 +-  0.92371812E-01
             Kinetic energy :  -0.13366041E+02 +-  0.92371811E-01
               Total energy :  -0.75000000E+01 +-  0.19039225E-09
                    Density :   0.13366041E+01 +-  0.92371811E-02
                Chi_thermal :  -0.48117746E+03 +-  0.74357874E+01
              Specific heat :   0.63000000E+04 +-  0.28949334E-06
  XX Ferro structure factor :   0.66339595E+00 +-  0.92371812E-02
  ZZ Ferro structure factor :   0.59979526E+00 +-  0.13397151E+00
     XX AF structure factor :   0.66339595E+00 +-  0.92371812E-02
  Root Mean Square of XX AF :   0.66744345E+00 +-  0.91678400E-02
     ZZ AF structure factor :   0.60697594E+00 +-  0.65949473E-01
  Root Mean Square of ZZ AF :   0.93838904E+00 +-  0.80170188E-01
           Potential energy :  -0.81697973E+00 +-  0.46185906E-01
             Hopping energy :   0.00000000E+00 +-  0.00000000E+00
           Double occupancy :   0.33660405E+00 +-  0.92371812E-02
      Magnetisation squared :   0.16584899E+00 +-  0.23092953E-02
============================================================================
 Mean Equal time Green's function:
point ; dx=  0 ; dy=  0 :       0.33169797E+00 +-  0.46185906E-02
point ; dx=  1 ; dy=  0 :       0.90913910E-19 +-  0.21767109E-19
point ; dx=  2 ; dy=  0 :       0.10248706E-17 +-  0.87345756E-18
point ; dx=  3 ; dy=  0 :       0.90668756E-19 +-  0.17605188E-19
point ; dx=  1 ; dy=  1 :       0.10470968E-18 +-  0.31097001E-19
point ; dx=  2 ; dy=  1 :       0.10156737E-18 +-  0.17494281E-19
point ; dx=  3 ; dy=  1 :       0.94415665E-19 +-  0.50506700E-19
point ; dx=  2 ; dy=  2 :       0.14505742E-18 +-  0.27613731E-19
point ; dx=  3 ; dy=  2 :       0.15492932E-18 +-  0.50843319E-19
point ; dx=  3 ; dy=  3 :       0.77725595E-18 +-  0.60334509E-18
============================================================================
 Up Equal time Green's function:
point ; dx=  0 ; dy=  0 :       0.32388726E+00 +-  0.63497263E-02
point ; dx=  1 ; dy=  0 :       0.91095237E-19 +-  0.49389213E-19
point ; dx=  2 ; dy=  0 :       0.18800745E-17 +-  0.17479869E-17
point ; dx=  3 ; dy=  0 :       0.67254951E-19 +-  0.53660565E-19
point ; dx=  1 ; dy=  1 :       0.65663400E-19 +-  0.37633479E-19
point ; dx=  2 ; dy=  1 :       0.99636357E-19 +-  0.33916387E-19
point ; dx=  3 ; dy=  1 :       0.81408744E-19 +-  0.82703274E-19
point ; dx=  2 ; dy=  2 :       0.14977815E-18 +-  0.39160085E-19
point ; dx=  3 ; dy=  2 :       0.18340730E-18 +-  0.11864303E-18
point ; dx=  3 ; dy=  3 :       0.12873625E-18 +-  0.62657571E-19
============================================================================
 Down Equal time Green's function:
point ; dx=  0 ; dy=  0 :       0.33950868E+00 +-  0.11324844E-01
point ; dx=  1 ; dy=  0 :       0.90732583E-19 +-  0.59107709E-19
point ; dx=  2 ; dy=  0 :       0.16966672E-18 +-  0.70045075E-19
point ; dx=  3 ; dy=  0 :       0.11408256E-18 +-  0.40446000E-19
point ; dx=  1 ; dy=  1 :       0.14375596E-18 +-  0.35523834E-19
point ; dx=  2 ; dy=  1 :       0.10349839E-18 +-  0.24027018E-19
point ; dx=  3 ; dy=  1 :       0.10742259E-18 +-  0.41372577E-19
point ; dx=  2 ; dy=  2 :       0.14033668E-18 +-  0.43342680E-19
point ; dx=  3 ; dy=  2 :       0.12645134E-18 +-  0.39002789E-19
point ; dx=  3 ; dy=  3 :       0.14257756E-17 +-  0.12150569E-17
============================================================================
 Density-density correlation fn: (up-up)
point ; dx=  0 ; dy=  0 :       0.66830203E+00 +-  0.46185906E-02
point ; dx=  1 ; dy=  0 :       0.44865889E+00 +-  0.66701745E-02
point ; dx=  2 ; dy=  0 :       0.44400760E+00 +-  0.69310982E-02
point ; dx=  3 ; dy=  0 :       0.44481686E+00 +-  0.67196425E-02
point ; dx=  1 ; dy=  1 :       0.44385470E+00 +-  0.78050077E-02
point ; dx=  2 ; dy=  1 :       0.44723882E+00 +-  0.46297183E-02
point ; dx=  3 ; dy=  1 :       0.44472900E+00 +-  0.55743170E-02
point ; dx=  2 ; dy=  2 :       0.45080016E+00 +-  0.66587102E-02
point ; dx=  3 ; dy=  2 :       0.44421114E+00 +-  0.63774693E-02
point ; dx=  3 ; dy=  3 :       0.44248299E+00 +-  0.77927711E-02
============================================================================
 Density-density correlation fn: (up-dn)
point ; dx=  0 ; dy=  0 :       0.33660405E+00 +-  0.92371812E-02
point ; dx=  1 ; dy=  0 :       0.44529754E+00 +-  0.55199144E-02
point ; dx=  2 ; dy=  0 :       0.44915449E+00 +-  0.58186900E-02
point ; dx=  3 ; dy=  0 :       0.44626181E+00 +-  0.62596733E-02
point ; dx=  1 ; dy=  1 :       0.44986727E+00 +-  0.47125912E-02
point ; dx=  2 ; dy=  1 :       0.44645907E+00 +-  0.84257216E-02
point ; dx=  3 ; dy=  1 :       0.44831835E+00 +-  0.80468954E-02
point ; dx=  2 ; dy=  2 :       0.44182242E+00 +-  0.61448935E-02
point ; dx=  3 ; dy=  2 :       0.44885830E+00 +-  0.64255861E-02
point ; dx=  3 ; dy=  3 :       0.44940392E+00 +-  0.11672579E-01
============================================================================
 XX Spin correlation function:
point ; dx=  0 ; dy=  0 :       0.66339595E+00 +-  0.92371812E-02
point ; dx=  1 ; dy=  0 :      -0.31095172E-36 +-  0.21895355E-36
point ; dx=  2 ; dy=  0 :       0.68966104E-36 +-  0.71236828E-36
point ; dx=  3 ; dy=  0 :      -0.12652949E-35 +-  0.10991088E-35
point ; dx=  1 ; dy=  1 :      -0.28075307E-36 +-  0.13225348E-36
point ; dx=  2 ; dy=  1 :      -0.59437774E-36 +-  0.46199351E-36
point ; dx=  3 ; dy=  1 :      -0.52304827E-36 +-  0.45512377E-36
point ; dx=  2 ; dy=  2 :       0.83842529E-37 +-  0.27590645E-36
point ; dx=  3 ; dy=  2 :      -0.10264715E-36 +-  0.22594839E-36
point ; dx=  3 ; dy=  3 :       0.28465948E-36 +-  0.31853494E-36
============================================================================
 ZZ Spin correlation function:
point ; dx=  0 ; dy=  0 :       0.66339595E+00 +-  0.92371812E-02
point ; dx=  1 ; dy=  0 :       0.67226961E-02 +-  0.27511146E-02
point ; dx=  2 ; dy=  0 :      -0.10293781E-01 +-  0.72579210E-02
point ; dx=  3 ; dy=  0 :      -0.28898851E-02 +-  0.49782888E-02
point ; dx=  1 ; dy=  1 :      -0.12025127E-01 +-  0.10793997E-01
point ; dx=  2 ; dy=  1 :       0.15594949E-02 +-  0.10666135E-01
point ; dx=  3 ; dy=  1 :      -0.71786887E-02 +-  0.11349613E-01
point ; dx=  2 ; dy=  2 :       0.17955477E-01 +-  0.44336633E-02
point ; dx=  3 ; dy=  2 :      -0.92943283E-02 +-  0.35255098E-02
point ; dx=  3 ; dy=  3 :      -0.13841861E-01 +-  0.27305371E-01
============================================================================
 Average Spin correlation function:
point ; dx=  0 ; dy=  0 :       0.66339595E+00 +-  0.92371812E-02
point ; dx=  1 ; dy=  0 :       0.22408987E-02 +-  0.91703820E-03
point ; dx=  2 ; dy=  0 :      -0.34312605E-02 +-  0.24193070E-02
point ; dx=  3 ; dy=  0 :      -0.96329502E-03 +-  0.16594296E-02
point ; dx=  1 ; dy=  1 :      -0.40083758E-02 +-  0.35979990E-02
point ; dx=  2 ; dy=  1 :       0.51983162E-03 +-  0.35553782E-02
point ; dx=  3 ; dy=  1 :      -0.23928962E-02 +-  0.37832044E-02
point ; dx=  2 ; dy=  2 :       0.59851589E-02 +-  0.14778878E-02
point ; dx=  3 ; dy=  2 :      -0.30981094E-02 +-  0.11751699E-02
point ; dx=  3 ; dy=  3 :      -0.46139537E-02 +-  0.91017903E-02
============================================================================
 Pairing correlation function:
point ; dx=  0 ; dy=  0 :       0.34689633E-09 +-  0.95196247E-11
point ; dx=  1 ; dy=  0 :      -0.24696641E-36 +-  0.14244730E-36
point ; dx=  2 ; dy=  0 :      -0.53008982E-37 +-  0.36191438E-36
point ; dx=  3 ; dy=  0 :      -0.45600518E-36 +-  0.42112562E-36
point ; dx=  1 ; dy=  1 :      -0.66705548E-37 +-  0.52270806E-37
point ; dx=  2 ; dy=  1 :       0.60015684E-37 +-  0.47436135E-37
point ; dx=  3 ; dy=  1 :      -0.20964835E-36 +-  0.19512241E-36
point ; dx=  2 ; dy=  2 :      -0.16661536E-36 +-  0.15618062E-36
point ; dx=  3 ; dy=  2 :      -0.14917642E-37 +-  0.14991652E-37
point ; dx=  3 ; dy=  3 :      -0.45143438E-36 +-  0.42819558E-36
============================================================================
 Pair-field correlation function - accumulated:
           S-Wave :                        NaN +-             NaN
          SX-Wave :                        NaN +-             NaN
           D-Wave :                        NaN +-             NaN
         SXX-Wave :                        NaN +-             NaN
         DXX-Wave :                        NaN +-             NaN
          PX-Wave :                        NaN +-             NaN
          PY-Wave :                        NaN +-             NaN
         PXY-Wave :                        NaN +-             NaN
         PYX-Wave :                        NaN +-             NaN
============================================================================
